guarddog/analyzer/sourcecode/code-execution.yml

```diff
@@ -41,6 +41,11 @@ rules:
           - pattern: subprocess.run([..., "... $ARG1 ...", ...], ...)
           - pattern: run([..., "$ARG1", ...], ...)
 
+          - pattern: subprocess.check_call($ARG1, ...)
+          - pattern: check_call($ARG1, ...)
+          - pattern: subprocess.check_call([..., "... $ARG1 ...", ...], ...)
+          - pattern: check_call([..., "$ARG1", ...], ...)
```

Comment:
I think you can get away the $ARG1, dropping the quotes.
Also, I think semgrep will complain the way it is

Reply:
It complains if I delete the `"... ..."` part, yes.  As-is, it's fine.